src/ex4/server/services/pokemon-utils.js

```diff
@@ -0,0 +1,74 @@
+export async function handleAddSingleOrMultiPokemonsTodo(model,pokemonClient, enterValue){
+
+    if(enterValue.includes(",")){
+        await handleAddMultiPokemonsTodo(model, pokemonClient, enterValue)
+    }
+    else {
+        await handleAddSinglePokemonTodo(model, pokemonClient, enterValue)
+    }
+}
+
+async function handleAddMultiPokemonsTodo(model, pokemonClient, enterValue){
```

Comment:
this method should already receive an array instead, and the parsing should be done outside (ex: inside handleAddSingleOrMultiPokemonsTodo)